scripts/e2e-deploy.sh

```diff
@@ -79,18 +70,15 @@ OC=$(which oc)
 OCDEPLOYER=$(which ocdeployer)
 IQE=$(which iqe)
 
-########## END CONFIGURATION ##########
-########## You shouldn't need to edit below here. ##########
-
-pushd ${E2E_REPO}
+pushd $E2E_REPO
 
 ### validation
-if [ -z "${REGISTRY_REDHAT_IO_SECRETS}" ]; then
+if [ -z "$REGISTRY_REDHAT_IO_SECRETS" ]; then
```

Comment:
Why is this being changed?  Both forms should be equivalent.

Reply:
I changed the script to use environment variables instead of setting the variables in the script and maybe it is a mac thing but in the `${REGISTRY_REDHAT_IO_SECRETS}` format - they weren't being picked up which prompted the change. I'm not a scripting expert so I'm definitely open to suggestions.

Reply:
Gotcha. Okay. If that makes it work better for you, I'm fine with it. They're *supposed* to be syntactically equivalent, with the curly braces being "safer" for use in complex strings - e.g. `FOO = "${somevar}_plus_other_stuff"`.

But at any rate, thanks for the extra info. :slightly_smiling_face: